### ISSUE TITLE
[Contribution] Missing Item Names - WIP complete item table (IDs and Names)

### ISSUE THREAD

**Issue by Exincracci** [lchmagKekse/TOTK-SaveGame-Editor#8]
Hello @lchmagKekse !

I took the liberty to collect some of the missing item names for your editor, since the BOTW ones are the same from _marcrobledo_. There are some that I couldn't get just yet because I'm doing this almost manually, because I lack knowledge to do it in a more efficent or practical way, but I believe it already provides some help.

Table: https://docs.google.com/spreadsheets/d/1EjGQIHpJX5hM3yKeOC9m8gDGgmtsTfNdUAh2NgidcwI/edit?usp=sharing

I'll keep you posted regarding updates I make on the file.

---
_**[EDIT] Table updated with information provided by @Karlos007  , and armor values pasted on lastest comment here.**_

**Comment by Karlos007** [comment]
<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/Carlo/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/Carlo/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">



</head>

<body link="#0563C1" vlink="#954F72">

New Weapons, Bows and Shields


String | Name
-- | --
Weapon_Sword_101 | Zonaite Sword
Weapon_Sword_103 | Wooden Stick
Weapon_Sword_105 | Boomerang (decayed)
Weapon_Sword_106 | Traveler's Sword (decayed)
Weapon_Sword_107 | Lizal Boomerang
Weapon_Sword_108 | Sturdy Wooden Stick
Weapon_Sword_109 | Gnarled Wooden Stick
Weapon_Sword_112 | Soldier's Broadsword (decayed)
Weapon_Sword_113 | Knight's Broadsword (decayed)
Weapon_Sword_114 | Eightfold Blade (decayed)
Weapon_Sword_124 | Royal Broadsword (decayed)
Weapon_Sword_125 | Forest Dweller's Sword  (decayed)
Weapon_Sword_127 | Zora Sword (decayed)
Weapon_Sword_129 | Gerudo Scimitar (decayed)
Weapon_Sword_131 | Feathered Edge (decayed)
Weapon_Sword_147 | Royal Guard's Sword (decayed)
Weapon_Sword_163 | Strong Zonaite Sword
Weapon_Sword_164 | Mighty Zonaite Sword
Weapon_Sword_166 | Gloom Sword
Weapon_Sword_167 | Tree Branch (Sky)
Weapon_Sword_168 | Wooden Stick (White)
Weapon_Lsword_101 | Zonaite Longsword
Weapon_Lsword_103 | Thick Stick
Weapon_Lsword_106 | Traveler's Claymore (decayed)
Weapon_Lsword_108 | Sturdy Thick Stick
Weapon_Lsword_109 | Gnarled Thick Stick
Weapon_Lsword_112 | Soldier's Claymore (decayed)
Weapon_Lsword_113 | Knight's Claymore (decayed)
Weapon_Lsword_114 | Eightfold Longblade (decayed)
Weapon_Lsword_124 | Royal Claymore (decayed)
Weapon_Lsword_127 | Zora Longsword (decayed)
Weapon_Lsword_129 | Gerudo Claymore (decayed)
Weapon_Lsword_136 | Cobble Crusher (decayed)
Weapon_Lsword_147 | Royal Guard's Claymore (decayed)
Weapon_Lsword_163 | Strong Zonaite Longsword
Weapon_Lsword_164 | Mighty Zonaite Longsword
Weapon_Lsword_166 | Gloom Club
Weapon_Lsword_168 | Thick Stick (White)
Weapon_Lsword_174 | Giant Boomerang (decayed)
Weapon_Spear_101 | Long Stick
Weapon_Spear_103 | Traveler's Spear (decayed)
Weapon_Spear_106 | Sturdy Long Stick
Weapon_Spear_108 | Gnarled Long Stick
Weapon_Spear_109 | Soldier's Spear (decayed)
Weapon_Spear_112 | Knight's Halberd (decayed)
Weapon_Spear_113 | Royal Halberd (decayed)
Weapon_Spear_124 | Forest Dweller's Spear (decayed)
Weapon_Spear_125 | Zora Spear (decayed)
Weapon_Spear_127 | Gerudo Spear (decayed)
Weapon_Spear_129 | Feathered Spear (decayed)
Weapon_Spear_132 | Royal Guard's Spear (decayed)
Weapon_Spear_147 | Strong Zonaite Spear
Weapon_Spear_163 | Mighty Zonaite Spear
Weapon_Spear_164 | Gloom Spear
Weapon_Spear_166 | Long Stick (White)
Weapon_Spear_168 | Throwing Spear (decayed)
Weapon_Spear_173 | Decayed Master Sword
Weapon_Bow_101 | Zonaite Bow
Weapon_Bow_104 | Construct Bow
Weapon_Bow_105 | Strong Construct Bow
Weapon_Bow_106 | Mighty Construct Bow
Weapon_Bow_107 | Old Wooden Bow
Weapon_Shield_101 | Zonaite Shield
Weapon_Shield_102 | Strong Zonaite Shield
Weapon_Shield_103 | Mighty Zonaite Shield
Weapon_Shield_107 | Old Wooden Shield



</body>

</html>

**Comment by Exincracci** [comment]
@lchmagKekse armor table from the spreadsheet link pasted below.

<html>
<body>
<google-sheets-html-origin><style type="text/css"></style>

String | Name
-- | --
Armor_1006_Head | Glide Mask
Armor_1006_Lower | Glide Tights
Armor_1006_Upper | Glide Shirt
Armor_1007_Head | Glide Mask ★
Armor_1007_Lower | Glide Tights ★
Armor_1007_Upper | Glide Shirt ★
Armor_1008_Head | Glide Mask ★★
Armor_1008_Lower | Glide Tights ★★
Armor_1008_Upper | Glide Shirt ★★
Armor_1009_Head | Glide Mask ★★★
Armor_1009_Lower | Glide Tights ★★★
Armor_1009_Upper | Glide Shirt ★★★
Armor_1010_Head | Glide Mask ★★★★
Armor_1010_Lower | Glide Tights ★★★★
Armor_1010_Upper | Glide Shirt ★★★★
Armor_1036_Head | Ancient Hero's Aspect
Armor_1036_Lower | MsgNotFound
Armor_1036_Upper | MsgNotFound
Armor_1037_Head | Ancient Hero's Aspect ★
Armor_1038_Head | Ancient Hero's Aspect ★★
Armor_1039_Head | Ancient Hero's Aspect ★★★
Armor_1040_Head | Ancient Hero's Aspect ★★★★
Armor_1041_Head | Hood of the Depths
Armor_1041_Lower | Gaiters of the Depths
Armor_1041_Upper | Tunic of the Depths
Armor_1042_Head | Hood of the Depths ★
Armor_1042_Lower | Gaiters of the Depths ★
Armor_1042_Upper | Tunic of the Depths ★
Armor_1043_Head | Hood of the Depths ★★
Armor_1043_Lower | Archaic Legwear
Armor_1043_Lower | Gaiters of the Depths ★★
Armor_1043_Upper | Archaic Tunic
Armor_1043_Upper | Tunic of the Depths ★★
Armor_1044_Head | Hood of the Depths ★★★
Armor_1044_Lower | Archaic Warm Greaves
Armor_1044_Lower | Gaiters of the Depths ★★★
Armor_1044_Upper | Tunic of the Depths ★★★
Armor_1045_Head | Royal Guard Cap
Armor_1045_Lower | Royal Guard Boots
Armor_1045_Upper | Royal Guard Uniform
Armor_1046_Head | Froggy Hood
Armor_1046_Head | Royal Guard Cap ★
Armor_1046_Lower | Froggy Leggings
Armor_1046_Lower | Royal Guard Boots ★
Armor_1046_Upper | Froggy Sleeve
Armor_1046_Upper | Royal Guard Uniform ★
Armor_1047_Head | Froggy Hood ★
Armor_1047_Head | Royal Guard Cap ★★
Armor_1047_Lower | Froggy Leggings ★
Armor_1047_Lower | Royal Guard Boots ★★
Armor_1047_Upper | Froggy Sleeve ★
Armor_1047_Upper | Royal Guard Uniform ★★
Armor_1048_Head | Froggy Hood ★★
Armor_1048_Head | Royal Guard Cap ★★★
Armor_1048_Lower | Froggy Leggings ★★
Armor_1048_Lower | Royal Guard Boots ★★★
Armor_1048_Upper | Froggy Sleeve ★★
Armor_1048_Upper | Royal Guard Uniform ★★★
Armor_1049_Lower | Froggy Hood ★★★
Armor_1049_Lower | Froggy Leggings ★★★
Armor_1049_Upper | Froggy Sleeve ★★★
Armor_1050_Head | Froggy Hood ★★★★
Armor_1050_Lower | Froggy Leggings ★★★★
Armor_1050_Upper | Froggy Sleeve ★★★★
Armor_1051_Head | Miner's Mask
Armor_1051_Lower | Miner's Trousers
Armor_1051_Upper | Miner's Top
Armor_1052_Head | Miner's Mask ★
Armor_1052_Lower | Miner's Trousers ★
Armor_1052_Upper | Miner's Top ★
Armor_1053_Head | Miner's Mask ★★
Armor_1053_Lower | Miner's Trousers ★★
Armor_1053_Upper | Miner's Top ★★
Armor_1054_Head | Miner's Mask ★★★
Armor_1054_Lower | Miner's Trousers ★★★
Armor_1054_Upper | Miner's Top ★★★
Armor_1055_Head | Miner's Mask ★★★★
Armor_1055_Lower | Miner's Trousers ★★★★
Armor_1055_Upper | Miner's Top ★★★★
Armor_1061_Head | Ember Headdress
Armor_1061_Lower | Ember Trousers
Armor_1061_Upper | Ember Shirt
Armor_1062_Head | Ember Headdress ★
Armor_1062_Lower | Ember Trousers ★
Armor_1062_Upper | Ember Shirt ★
Armor_1063_Head | Ember Headdress ★★
Armor_1063_Lower | Ember Trousers ★★
Armor_1063_Upper | Ember Shirt ★★
Armor_1064_Head | Ember Headdress ★★★
Armor_1064_Lower | Ember Trousers ★★★
Armor_1064_Upper | Ember Shirt ★★★
Armor_1065_Head | Ember Headdress ★★★★
Armor_1065_Lower | Ember Trousers ★★★★
Armor_1065_Upper | Ember Shirt ★★★★
Armor_1066_Head | Charged Headdress
Armor_1066_Lower | Charged Trousers
Armor_1066_Upper | Charged Shirt
Armor_1067_Head | Charged Headdress ★
Armor_1067_Lower | Charged Trousers ★
Armor_1067_Upper | Charged Shirt ★
Armor_1068_Head | Charged Headdress ★★
Armor_1068_Lower | Charged Trousers ★★
Armor_1068_Upper | Charged Shirt ★★
Armor_1069_Head | Charged Headdress ★★★
Armor_1069_Lower | Charged Trousers ★★★
Armor_1069_Upper | Charged Shirt ★★★
Armor_1070_Head | Charged Headdress ★★★★
Armor_1070_Lower | Charged Trousers ★★★★
Armor_1070_Upper | Charged Shirt ★★★★
Armor_1071_Head | Frostbite Headdress
Armor_1071_Lower | Frostbite Trousers
Armor_1071_Upper | Frostbite Shirt
Armor_1072_Head | Frostbite Headdress ★
Armor_1072_Lower | Frostbite Trousers ★
Armor_1072_Upper | Frostbite Shirt ★
Armor_1073_Head | Frostbite Headdress ★★
Armor_1073_Lower | Frostbite Trousers ★★
Armor_1073_Upper | Frostbite Shirt ★★
Armor_1074_Head | Frostbite Headdress ★★★
Armor_1074_Lower | Frostbite Trousers ★★★
Armor_1074_Upper | Frostbite Shirt ★★★
Armor_1075_Head | Frostbite Headdress ★★★★
Armor_1075_Lower | Frostbite Trousers ★★★★
Armor_1075_Upper | Frostbite Shirt ★★★★
Armor_1076_Head | Cece Hat
Armor_1086_Head | Mystic Headpiece
Armor_1086_Lower | Mystic Trousers
Armor_1086_Upper | Mystic Robe
Armor_1091_Head | Zonaite Helm
Armor_1091_Lower | Zonaite Shin Guards
Armor_1091_Upper | Zonaite Waistguard
Armor_1092_Head | Zonaite Helm ★
Armor_1092_Lower | Zonaite Shin Guards ★
Armor_1092_Upper | Zonaite Waistguard ★
Armor_1093_Head | Zonaite Helm ★★
Armor_1093_Lower | Zonaite Shin Guards ★★
Armor_1093_Upper | Zonaite Waistguard ★★
Armor_1094_Head | Zonaite Helm ★★★
Armor_1094_Lower | Zonaite Shin Guards ★★★
Armor_1094_Upper | Zonaite Waistguard ★★★
Armor_1095_Head | Zonaite Helm ★★★★
Armor_1095_Lower | Zonaite Shin Guards ★★★★
Armor_1095_Upper | Zonaite Waistguard ★★★★
Armor_1096_Head | Mask of Awakening
Armor_1096_Lower | Trousers of Awakening
Armor_1096_Upper | Tunic of Awakening
Armor_1097_Head | Mask of Awakening ★
Armor_1097_Lower | Trousers of Awakening ★
Armor_1097_Upper | Tunic of Awakening ★
Armor_1098_Head | Mask of Awakening ★★
Armor_1098_Lower | Trousers of Awakening ★★
Armor_1098_Upper | Tunic of Awakening ★★
Armor_1099_Head | Mask of Awakening ★★★
Armor_1099_Lower | Trousers of Awakening ★★★
Armor_1099_Upper | Tunic of Awakening ★★★
Armor_1100_Head | Mask of Awakening ★★★★
Armor_1100_Lower | Trousers of Awakening ★★★★
Armor_1100_Upper | Tunic of Awakening ★★★★
Armor_1106_Upper | Champion's Leather
Armor_1107_Upper | Champion's Leather ★
Armor_1108_Upper | Champion's Leather ★★
Armor_1109_Upper | Champion's Leather ★★★
Armor_1110_Upper | Champion's Leather ★★★★
Armor_1125_Head | Horriblin Mask
Armor_1145_Head | Hood of the Depths ★★★★
Armor_1145_Lower | Gaiters of the Depths ★★★★
Armor_1145_Upper | Tunic of the Depths ★★★★
Armor_1149_Head | Royal Guard Cap ★★★★
Armor_1149_Lower | Royal Guard Boots ★★★★
Armor_1149_Upper | Royal Guard Uniform ★★★★
Armor_1151_Head | Well-Worn Hair Bond
Armor_1156_Head | Hylian Hood ★★★★
Armor_116_Upper | Tunic of Memories ★★★★
Armor_1300_Head | Yiga Mask
Armor_1300_Lower | Yiga Tights
Armor_1300_Upper | Yiga Armor
Armor_1301_Head | Yiga Mask ★
Armor_1301_Lower | Yiga Tights ★
Armor_1301_Upper | Yiga Armor ★
Armor_1302_Head | Yiga Mask ★★
Armor_1302_Lower | Yiga Tights ★★
Armor_1302_Upper | Yiga Armor ★★
Armor_1303_Head | Yiga Mask ★★★
Armor_1303_Lower | Yiga Tights ★★★
Armor_1303_Upper | Yiga Armor ★★★
Armor_1304_Head | Yiga Mask ★★★★
Armor_1304_Lower | Yiga Tights ★★★★
Armor_1304_Upper | Yiga Armor ★★★★
Armor_151_Upper | Tunic of Memories ★★★★
Armor_999_Head | MsgNotFound


</body>
</html>

**Comment by lchmagKekse** [comment]
I added the item names. Thanks for your contribution. @Karlos007 @Exincracci

**Comment by Cr4nkSt4r** [comment]
I'll leave the savegame, I posted on marcrobledo repo, here too.
[Save 3 = Zestiva - Perfect Save + Epona.zip](https://github.com/lchmagKekse/TOTK-SaveGame-Editor/files/11445931/Save.3.Zestiva.-.Perfect.Save.%2B.Epona.zip)
credits to zestiva from gbatemp

**Information for the savegame:**
money max
40 heart
stamina max
battery max
master sword
ganondorf bow
twilight bow
hylian shield
all 93 armor and all star level 4 max
all 248 material * 999
all 60 cook
all 27 zonau device * 999
all 11 horse saddle
all 37 substitute cloth
all collection item * 999
all 119 dungeon temple & 120 tree root open

**Comment by Exincracci** [comment]
> I'll leave the savegame, I posted on marcrobledo repo, here too. [Save 3 = Zestiva - Perfect Save + Epona.zip](https://github.com/lchmagKekse/TOTK-SaveGame-Editor/files/11445931/Save.3.Zestiva.-.Perfect.Save.%2B.Epona.zip) credits to zestiva from gbatemp
> 
> **Information for the savegame:** money max 40 heart stamina max battery max master sword ganondorf bow twilight bow hylian shield all 93 armor and all star level 4 max all 248 material * 999 all 60 cook all 27 zonau device * 999 all 11 horse saddle all 37 substitute cloth all collection item * 999 all 119 dungeon temple & 120 tree root open

@Cr4nkSt4r thank you for that. Due to this file, I was able to update a table containing the IDs/names of important items here[here](https://github.com/lchmagKekse/TOTK-SaveGame-Editor/issues/5#issuecomment-1542830506)

**Comment by Exincracci** [comment]
@lchmagKekse I tested the editor upgrades you made so far, and noticed that some items were display with only the ID in red. I realized that [my table](https://docs.google.com/spreadsheets/d/1EjGQIHpJX5hM3yKeOC9m8gDGgmtsTfNdUAh2NgidcwI/) was containing some armor names with incorrect IDs (set Of the Depths, Lowered version of Hylian Hood and Royal Guard set that I remember now). My apologies for that. Now I believe you won't have a problem with missing things.
